ajax/libs/vega/package.json

```diff
@@ -13,8 +13,13 @@
     {
       "basePath": "",
       "files": [
-        "vega.js",
-        "vega.min.js"
+        "vega*.(js|css|map)"
```

Comment:
Sorry that I didn't note that the syntax is wrong, should be `vega*.+(js|css|map)"` (needs **+**)